### ISSUE TITLE
CreateProposal View

### ISSUE THREAD

**Issue by evgenibir** [hypha-dao/hypha-web#83]
## Blocked by

- Missing design

## TODO

- [ ] @massimobergero Select Proposal Outcome

## Visual
<img width="499" alt="Image" src="https://github.com/user-attachments/assets/f609d7db-29ae-44f3-918c-03d89571f42c" />

### Proposal Outcome

[Link to prototype](https://www.figma.com/proto/vsi2L9fU0EGFoxY2byPUl5/Hypha-%2B-Radix-v3-1.0?page-id=3423%3A5672&node-id=5758-28841&viewport=2108%2C-2073%2C0.92&t=vRwlrspHvc6vplmc-1&scaling=scale-down&content-scaling=fixed&starting-point-node-id=3977%3A21249&show-proto-sidebar=1)

<img width="1442" alt="Image" src="https://github.com/user-attachments/assets/e023900c-1cb8-4cba-980a-fcb373b0c65b" />

<img width="1445" alt="Image" src="https://github.com/user-attachments/assets/3a9863aa-4c56-4dc4-b36a-979066b81702" />

**Comment by massimobergero** [comment]
@plitzenberger Exactly, or you can flag "skip discussion phase" and jump directly to the proposal creation panel
